### DISCUSSION TITLE
Teensy 4 / 4.1 with SavvyCAN

### DISCUSSION THREAD

**Discussion by Doig5710** [collin80/SavvyCAN#474] [Q&A]
I know theres the TeensyRET for the older 3.6 boards but due to the chip shortage they all apper to be sold out for sometime now, i have a 4.1 but unable to load it. Is there any chance of a update oneday to allow a 4 or 4.1 to work with savvycan in the future?

**Comment by Evautos** [comment]
I would like that too.
